pykafka/producer.py

```diff
@@ -120,6 +105,11 @@ def _send_request(self, broker, req, attempt):
         :type req: :class:`pykafka.protocol.ProduceRequest`
         :param attempt: The current attempt count. Used for retry logic
         :type attempt: int
+        :param q_info: A QueueInfo instance containing information
+            about the queue from which the messages in `req` were removed.
+            Only used when called from the
+            :class:`pykafka.producer.AsyncProducer`
```

Comment:
If this is only relevant when used from `AsyncProducer` why is it in `Producer`?  This smells like something where Async should just have its own implementation instead of trying super hard to share code.